docs/site/express-with-lb4-rest-tutorial.md

```diff
@@ -0,0 +1,358 @@
+---
+lang: en
+title: 'Creating an Express Application with LoopBack REST API'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/express-with-lb4-rest-tutorial.html
+summary: A simple Express application with LoopBack 4 REST API
+---
+
+## Overview
+
+[Express](https://expressjs.com) is an unopinionated Node.js framework. LoopBack
+REST API can be mounted to an Express application and be used as middleware.
+This way the user can mix and match features from both frameworks to suit their
+needs.
+
+This tutorial assumes familiarity with scaffolding a LoopBack 4 application,
+[`Models`](Model.md), [`DataSources`](DataSources.md),
+[`Repositories`](Repositories.md), and [`Controllers`](Controllers.md). To see
+how they're used in a LoopBack application, please see the
+[`Todo` tutorial](todo-tutorial.md).
+
+## Try it out
+
+If you'd like to see the final results of this tutorial as an example
+application, follow these steps:
+
+1.  Run the `lb4 example` command to select and clone the express-composition
+    repository:
+
+    ```sh
+    $ lb4 example
+    ? What example would you like to clone? (Use arrow keys)
+    todo: Tutorial example on how to build an application with LoopBack 4.
+    todo-list: Continuation of the todo example using relations in LoopBack 4.
+    hello-world: A simple hello-world Application using LoopBack 4.
+    log-extension: An example extension project for LoopBack 4.
+    rpc-server: A basic RPC server using a made-up protocol.
+    > express-composition: A simple Express application that uses LoopBack 4 REST API.
+    ```
+
+2.  Switch to the directory.
+
+    ```sh
+    cd loopback4-example-express-composition
+    ```
+
+3.  Finally, start the application!
+
+    ```sh
+    $ npm start
+
+    Server is running at http://127.0.0.1:3000
+    ```
+
+### Create your LoopBack Application
+
+#### Scaffold your Application
```

Comment:
For the record: I'm wondering if we should omit the steps to build up a `Note` application with LoopBack 4 (we already have tutorial for that) and go directly to `Express Class`.

Reply:
+1 on just showing the delta 
Perhaps use `todo` application (don't think we have `Note` application)?

Reply:
@raymondfeng I tried to omit the explanations and only put the CLI parts, but I can try to trim that down even further (e.g. create a `Note` repository with `DsDataSource` and the `Note` model). What do you think?

@dhmlau we had a discussion above about using the `todo` application, but ultimately decided to use a new, simple `Note` one (see https://github.com/strongloop/loopback-next/pull/2306#issuecomment-459277370 and its replies).

Reply:
What I meant was to use todo app as the starting point to build this tutorial, and we just need to show the additional steps required to mount Express.